src/tree/build-tree-test.cc

```diff
@@ -88,7 +88,7 @@ void TestGenRandStats() {
 
 
 void TestBuildTree() {
-  for (int32 p = 0; p < 3; p++) {
+  for (int32 p = 0; p < 30; p++) {
```

Comment:
please make sure this doesn't take too long (e.g. more than 0.1 second).. I may have set it to a small number because it was slow.